cli/inagoctl.go

```diff
@@ -34,19 +38,34 @@ var (
 			// This callback is executed after flags are parsed and before any
 			// command runs.
 
+			logLevel := "INFO"
+			if globalFlags.Verbose {
+				logLevel = "DEBUG"
+			}
+
+			logger := requestcontext.MustGetLogger(
+				requestcontext.LoggerConfig{
+					Name:                "inago",
+					Level:               logLevel,
+					Color:               isatty.IsTerminal(os.Stderr.Fd()),
+					IncludeNameInFormat: false,
+				},
+			)
+			newLogger = &logger
```

Comment:
`cannot take the address of requestcontext.MustGetLogger(requestcontext.LoggerConfig literal)`

in go, you can't take the address of a function call, you need to assign it to a variable first (which you can then take the address of)